pixelhouse/transform/simple.py

```diff
@@ -32,6 +32,12 @@ def draw(self, cvs, t=0.0):
 
 
 class rotate(Artist):
+
+    """ Rotates a canvas by a given amount.
+        I hope you like radians.
```

Comment:
:stuck_out_tongue: but it might make sense to have a version that can take degrees too!